### ISSUE TITLE
Make IDM.gov Accessibility Fixes

### ISSUE THREAD

**Issue by JillTunick** [GSA/idmanagement.gov#193]
Make changes to remediate green-coded accessibility violations logged in the May 22 IDM.gov Prod Report Module Violations and Solutions spreadsheet (https://docs.google.com/spreadsheets/d/1XD4R1XIv5nuobYTpD9dU09zb20FKFHyi/edit#gid=273438484) and make changes to fix assorted punctuation, capitalization, consistency, and other issues discovered during manual accessibility testing on IDM.gov.
